hypothesis-python/src/hypothesis/internal/compat.py

```diff
@@ -145,30 +145,31 @@ def update_code_location(code, newfile, newlineno):
         # added to facilitate future-proof code.  See BPO-37032 for details.
         return code.replace(co_filename=newfile, co_firstlineno=newlineno)
 
-    # This field order is accurate for 3.5 - 3.7, but not 3.8 when a new field
-    # was added for positional-only arguments.  However it also added a .replace()
-    # method that we use instead of field indices, so they're fine as-is.
-    CODE_FIELD_ORDER = [
-        "co_argcount",
-        "co_kwonlyargcount",
-        "co_nlocals",
-        "co_stacksize",
-        "co_flags",
-        "co_code",
-        "co_consts",
-        "co_names",
-        "co_varnames",
-        "co_filename",
-        "co_name",
-        "co_firstlineno",
-        "co_lnotab",
-        "co_freevars",
-        "co_cellvars",
-    ]
-    unpacked = [getattr(code, name) for name in CODE_FIELD_ORDER]
-    unpacked[CODE_FIELD_ORDER.index("co_filename")] = newfile
-    unpacked[CODE_FIELD_ORDER.index("co_firstlineno")] = newlineno
-    return type(code)(*unpacked)
+    else:  # pragma: no cover
```

Comment:
I've put the following code inside an `else` block, because I'm not aware of any better way to mark the whole thing as uncovered.